src/weekly/WeeklyExporter.js

```diff
@@ -147,3 +147,44 @@ export class CaHandDeliveryExporter extends CaExporter {
     return handDeliveryValue.trim().toUpperCase() === 'YES'
   }
 }
+
+const euCountries: string[] = [
+  'Austria',
+  'Belgium',
+  'Bulgaria',
+  'Croatia',
+  'Cyprus',
+  'Czech Republic',
```

Comment:
Czech Republic have updated their name to Czechia so we may want to add both

Reply:
It isn't in Zuora's list of countries but I'll add it in case that changes.  
Although, according to [this](https://unterm.un.org/UNTERM/Display/Record/UNHQ/NA/4275087d-4018-4082-899d-95f37efeda65), Czechia is just the short name.

There is a general issue about keeping this list up to date.  Ideally we would query an API for the list somewhere but then we would also have to cross-reference it with Zuora's list so that will probably be a future optimisation after this repo has been rewritten in Scala.

Reply:
That makes sense, hopefully EU names are fairly static but should any new member states join we will have this manual step